### PR TITLE
Translate to portuguese/br

### PR DESCRIPTION
**Pull request by kivervinicius**
I translated only half of the file, but the future will end.

### CODE REVIEW

File: pt/tutorials-and-examples/blog/blog.rst [hunk 1]
```diff
@@ -1,54 +1,58 @@
-Blog Tutorial
+Construindo um blog
 #############
 
-This tutorial will walk you through the creation of a simple blog application.
-We'll be installing CakePHP, creating a database, and creating enough
-application logic to list, add, edit, and delete blog posts.
+Este tutorial irá orientá-lo através da criação de um aplicativo simples blog.
```
Comment (andtxr) [inline pt/tutorials-and-examples/blog/blog.rst]: This conjuncture between words should be improved.
_Este tutorial irá orientá-lo através da criação de uma simples aplicação de blog._

File: pt/tutorials-and-examples/blog/blog.rst [hunk 2]
```diff
@@ -1,54 +1,58 @@
...
-   tutorial. You'll need to know enough about SQL in order to create a
-   database: CakePHP will be taking the reins from there. Since we're using MySQL,
-   also make sure that you have ``pdo_mysql`` enabled in PHP.
-#. Basic PHP knowledge.
+#. Um servidor web rodando. Vamos supor que você está usando o Apache, 
+   embora as instruções para usar outros servidores sejam muito semelhantes. 
+   Poderíamos ter que brincar um pouco com a configuração do servidor, 
+   mas a maioria das pessoas pode ter o CakePHP instalado e funcionando sem 
+   qualquer configuração em tudo. 
+   Certifique-se de que você tem o PHP 5.4.16 ou superior, e que os ``mbstring`` e ``intl`
```
Comment (andtxr) [inline pt/tutorials-and-examples/blog/blog.rst]: Here you are missing the markdown close for ``intl``.

File: pt/tutorials-and-examples/blog/blog.rst [hunk 3]
```diff
@@ -1,54 +1,58 @@
-Blog Tutorial
+Construindo um blog
 #############
```
Comment (andtxr) [inline pt/tutorials-and-examples/blog/blog.rst]: Title length must match the underline

Comment (andtxr) [inline pt/tutorials-and-examples/blog/blog.rst]: Title length must match the underline

File: pt/tutorials-and-examples/blog/blog.rst [hunk 4]
```diff
@@ -1,54 +1,58 @@
...
+   vai tomar as rédeas a partir daí. Desde que nós estamos usando o MySQL, 
+   também certificar-se de que você ``pdo_mysql`` habilitado no PHP.
+#. Conhecimento básico de PHP.
 
-Let's get started!
+Vamos começar!
 
-Getting CakePHP
+Instalando o CakePHP
 ===============
```
Comment (andtxr) [inline pt/tutorials-and-examples/blog/blog.rst]: Title length must match the underline

Comment (andtxr) [inline pt/tutorials-and-examples/blog/blog.rst]: You are welcome!
Thanks for helping @kivervinicius :+1:

**Comment by andtxr** [comment]
You are welcome!
Thanks for helping @kivervinicius :+1:

**Comment by kivervinicius** [comment]
:+1:

File: pt/tutorials-and-examples/blog/blog.rst (hunk 3 above)
Comment (kivervinicius) [inline pt/tutorials-and-examples/blog/blog.rst]: His suggestion is to remove the 'um'?

File: pt/tutorials-and-examples/blog/blog.rst (hunk 4 above)
Comment (kivervinicius) [inline pt/tutorials-and-examples/blog/blog.rst]: His suggestion is to remove the 'o'?

File: pt/tutorials-and-examples/blog/blog.rst (hunk 3 above)
Comment (andtxr) [inline pt/tutorials-and-examples/blog/blog.rst]: No, the underline length should match the char length of the title.
It is harder to detect in Github editor due the editor font.
Ex: 19 characters, 19 #

Construindo um blog
###################

Comment (kivervinicius) [inline pt/tutorials-and-examples/blog/blog.rst]: Truth, of course, sorry but I'm noob in markdown

File: pt/tutorials-and-examples/blog/blog.rst [hunk 5]
```diff
@@ -1,54 +1,58 @@
...
+
+Aqui está o que você vai precisar:
+
+#. Um servidor web rodando. Vamos supor que você está usando o Apache, 
+   embora as instruções para usar outros servidores sejam muito semelhantes. 
+   Poderíamos ter que brincar um pouco com a configuração do servidor, 
+   mas a maioria das pessoas pode ter o CakePHP instalado e funcionando sem 
+   qualquer configuração em tudo. 
+   Certifique-se de que você tem o PHP 5.4.16 ou superior, e que os ``mbstring`` e ``intl`` 
+   extensões são ativadas no PHP.
```
Comment (andtxr) [inline pt/tutorials-and-examples/blog/blog.rst]: _e que as extensões ``mbstring`` e ``intl`` estão ativadas no PHP_

File: pt/tutorials-and-examples/blog/blog.rst [hunk 6]
```diff
@@ -1,54 +1,58 @@
...
+#. Um servidor de banco de dados. Nós vamos estar usando servidor MySQL neste tutorial.
```
Comment (andtxr) [inline pt/tutorials-and-examples/blog/blog.rst]: _Nós iremos utilizar um servidor MySQL neste tutorial._
You are free to adapt the translation pro readability.

File: pt/tutorials-and-examples/blog/blog.rst [hunk 7]
```diff
@@ -1,54 +1,58 @@
...
+#. Um servidor web rodando. Vamos supor que você está usando o Apache, 
+   embora as instruções para usar outros servidores sejam muito semelhantes. 
+   Poderíamos ter que brincar um pouco com a configuração do servidor, 
+   mas a maioria das pessoas pode ter o CakePHP instalado e funcionando sem 
+   qualquer configuração em tudo. 
+   Certifique-se de que você tem o PHP 5.4.16 ou superior, e que os ``mbstring`` e ``intl`` 
+   extensões são ativadas no PHP.
+#. Um servidor de banco de dados. Nós vamos estar usando servidor MySQL neste tutorial. 
+   Você precisa saber o suficiente sobre SQL para criar um banco de dados: CakePHP 
+   vai tomar as rédeas a partir daí. Desde que nós estamos usando o MySQL,
```
Comment (andtxr) [inline pt/tutorials-and-examples/blog/blog.rst]: _Desde que nós estejamos usando o MySQL_

**Comment by andtxr** [comment]
I think it is better to do a full text review before merging. :smile:

**Comment by kivervinicius** [comment]
Ok, sorry for this, i'm check this before.

**Comment by andtxr** [comment]
Due this kind of situation I am starting a try to unify brazilian translators through @cakephp-br (http://cakephp-br.org/documentacao). This way I expect to have a better review flow before pull requests.

**Comment by andtxr** [comment]
I am already working on this translation. You may close it. :+1:

**Pull request closed without merging**